### ISSUE TITLE
AppleMiniaturizeOnDoubleClick Isn't working

### ISSUE THREAD

**Issue by kalvish21** [indragiek/INAppStoreWindow#204]
I'm not able to minimize when I double click the window. I have El Capitan and the setting is ON (in El Capitan the setting is Preferences > Dock > "Double click a window's title bar to..". I have it set to minimize. The method below when I debug returns false for the minimize:

```
- (void)mouseUp:(NSEvent *)theEvent
{
	if ([theEvent clickCount] == 2) {
		// Get settings from "System Preferences" >  "Appearance" > "Double-click on windows title bar to minimize"
		NSString *const MDAppleMiniaturizeOnDoubleClickKey = @"AppleMiniaturizeOnDoubleClick";
		NSUserDefaults *userDefaults = [NSUserDefaults standardUserDefaults];
		BOOL shouldMiniaturize = [[userDefaults objectForKey:MDAppleMiniaturizeOnDoubleClickKey] boolValue];
		if (shouldMiniaturize) {
			[[self window] performMiniaturize:self];
		}
	}
}
```

I think the preference **AppleMiniaturizeOnDoubleClick** is incorrect?

**Comment by kalvish21** [comment]
I figured it out. In OS X 10.11 Apple changed the setting. I will create a pull request for this.

```
	if ([theEvent clickCount] == 2) {
		// Get settings from "System Preferences" >  "Appearance" > "Double-click on windows title bar to minimize"
		NSString *const MDAppleMiniaturizeOnDoubleClickKey = @"AppleMiniaturizeOnDoubleClick";
        NSString *const MDAppleMiniaturizeOnDoubleClickKeyElCapitan = @"AppleActionOnDoubleClick";
		NSUserDefaults *userDefaults = [NSUserDefaults standardUserDefaults];
		BOOL shouldMiniaturize = [[userDefaults objectForKey:MDAppleMiniaturizeOnDoubleClickKey] boolValue] || [[userDefaults objectForKey:MDAppleMiniaturizeOnDoubleClickKeyElCapitan] isEqualToString:@"Minimize"];
		if (shouldMiniaturize) {
			[[self window] performMiniaturize:self];
		}
	}
```

**Comment by indragiek** [comment]
Fixed in #205
